packages/components/README.md

```diff
@@ -59,6 +41,27 @@ export function DynamicSquiggleChart({ squiggleString }) {
   }
 }
 
+# Usage in a `react` project (future functionality).
+
+Right now, this functionality is [broken](https://github.com/quantified-uncertainty/squiggle/issues/1234), but we are aiming to fix it.
+
+For example, in a fresh `create-react-app` project
```

Comment:
New React docs (https://react.dev) don't mention create-react-app, and the current plan for it is to turn it into a launcher for recommended solutions (such as Next.js): https://github.com/reactjs/react.dev/pull/5487#issuecomment-1409720741

So we probably shouldn't mention CRA at all and recommended that users go with Next.js or [some other framework](https://react.dev/learn/start-a-new-react-project#production-grade-react-frameworks).

Suggestion:
- replace `create-react-app` with "Next.js" in the line above
- replace `App.js` below with `pages/index.js`
- remove the link to #1234, squiggle-components work fine with Next.js